forte/ontology_specs/ag_news.json

```diff
@@ -4,11 +4,11 @@
       "base_ontology.json"
     ],
     "additional_prefixes": [
-        "ftx"
+        "ftx.news"
     ],
     "definitions": [
       {
-        "entry_name": "ftx.onto.ag_news.Description",
+        "entry_name": "ftx.news.ag_news.Description",
```

Comment:
probably no need to create a subpackage name for each example, we can put them all under `ftx.onto`?